src/qp/quant_pdf.py

```diff
@@ -193,12 +193,13 @@ def _instantiate_pdf_constructor(self):
             self._quants, self._locs)
 
 
-    def _pdf(self, x, row):
+    def _pdf(self, x, *args):
```

Comment:
What was the pylint rule that it was complaining about here?

Reply:
Signature didn't match base class.